tools/Google.Events.Tools.CodeGenerator/Program.cs

```diff
@@ -99,9 +99,18 @@ void GenerateCloudEventTypes()
                         writer.WriteLine("    {");
                         foreach (var type in group.Select(ce => ce.Type))
                         {
-                            var suffix = type.Split('.').Last();
+                            // Firestore and Datastore have separate "withAuthContext" variants; in those cases we want to
+                            // use the previous segment as well. Hard-coding this isn't pleasant, but it'll do until we see
+                            // anythign else we need to do.
```

Comment:
tiny typo: anythign => anything

Reply:
Whoops, thanks - will fix in the morning.